tests/unit/test-loader-test.js

```diff
@@ -121,7 +121,7 @@ test('throws an error if partition isn\'t a number', function(assert) {
 
   assert.throws(() => {
     this.TestLoader.load();
-  }, /You must specify numbers for partition \(you specified 'foo'\)/);
+  }, /You must specify numbers for partition/);
```

Comment:
Rather than complicate things and try to keep a copy of the original `partition` before parsing I decided to remove the `(you specified ..)` part. Happy to add it back in if you think it should stay.